settings.gradle.kts

```diff
@@ -9,6 +9,13 @@ dependencyResolutionManagement {
     repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
     repositories {
         mavenCentral()
+        google {
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+            }
```

Comment:
Is there a problem/conflict including the entire google maven repo (removing the constraints)?

Reply:
Nope. Add these regex will speed up downloading.

Reply:
By how much? I lean toward the simpler change, but I could be convinced if it was a big perf swing.

Reply:
No much effects for small projects. I can revert this if you want.